src/storm/solver/helper/ValueIterationOperator.h

```diff
@@ -291,35 +291,41 @@ class ValueIterationOperator {
                         OperandType const& operand, OffsetType const& offsets, uint64_t offsetIndex) const {
         STORM_LOG_ASSERT(*matrixColumnIt >= StartOfRowIndicator, "VI Operator in invalid state.");
         auto result{robustInitializeRowRes<RobustDirection>(operand, offsets, offsetIndex)};
-        std::vector<std::pair<SolutionType, SolutionType>> tmp;  // TODO this reallocation is too costly.
+        static std::vector<std::pair<SolutionType, SolutionType>> robustOrder;
```

Comment:
Can we avoid having this `static`?  I see how the cached memory is helpful, but maybe use a private member like this:
```
template <typename VT>
struct ApplyCache {};

template<>
struct ApplyCache <storm::Interval>
{
    std::vector<std::pair<SolutionType, SolutionType>> robustOrder;
};

ApplyCache<ValueType> applyCache;
```

This way, there is no robustOrder for Operands that are not using intervals.